docs/CHANGELOG.asciidoc

```diff
@@ -54,6 +54,12 @@ To review the breaking changes in previous versions, refer to the following:
 
 {kibana-ref-all}/8.2/release-notes-8.2.0.html#breaking-changes-8.2.0[8.2.0] | {kibana-ref-all}/8.1/release-notes-8.1.0.html#breaking-changes-8.1.0[8.1.0] | {kibana-ref-all}/8.0/release-notes-8.0.0.html#breaking-changes-8.0.0[8.0.0] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc2.html#breaking-changes-8.0.0-rc2[8.0.0-rc2] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc1.html#breaking-changes-8.0.0-rc1[8.0.0-rc1] | {kibana-ref-all}/8.0/release-notes-8.0.0-beta1.html#breaking-changes-8.0.0-beta1[8.0.0-beta1] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha2.html#breaking-changes-8.0.0-alpha2[8.0.0-alpha2] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha1.html#breaking-changes-8.0.0-alpha1[8.0.0-alpha1]
 
+[discrete]
+=== Known issues
+
+* Elasticsearch API commands will fail if you add any URL arguments (e.g. `?v`, `?pretty`) to the call. 
```

Comment:
I'd suggest changing this to:

```
* API requests will fail if you add any URL arguments (e.g. `?v`, `?pretty`) to the request.
```

Let's standardize on the term "request" instead of "call" or "command". Also, I left out "Elasticsearch" because Console now supports Kibana APIs too, and I'm pretty sure the bug extends to those as well.

Reply:
Done, thx!